testsuite/integration/clustering/pom.xml

```diff
@@ -45,7 +45,7 @@
         <surefire.forked.process.timeout>5400</surefire.forked.process.timeout>
         <test-group>org/jboss/as/test/clustering/cluster</test-group>
         <version.org.jboss.byteman>4.0.6</version.org.jboss.byteman>
-        <version.org.infinispan.server>11.0.3.Final</version.org.infinispan.server>
+        <version.org.infinispan.server>${version.org.infinispan}</version.org.infinispan.server>
```

Comment:
This essentially removes a fix for https://issues.redhat.com/browse/JBEAP-20157 - please don't do that, just bump the version here.

Reply:
Not really.  The version for version of the server can be set independently.

Reply:
Yes, but that would require every prod build to be updated and kept in sync which is not feasible.